jcl/src/openj9.gpu/share/classes/com/ibm/gpu/Version.java

```diff
@@ -27,7 +27,15 @@
  * The main method can be launched from the command line, which will print
  * the current level to stdout.
  */
+/*[IF Java16]*/
```

Comment:
Same question about formatting as above.  I won't repeat it here but we should do the same with all copies of the pattern